catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/cache/solr/impl/CachingFederationStrategy.java

```diff
@@ -547,13 +547,7 @@ private QueryRequest getQueryRequest() {
 
     private SourceResponse cloneResponse(SourceResponse sourceResponse) {
       List<Result> clonedResults =
-          sourceResponse
-              .getResults()
-              .stream()
-              .map(Result::getMetacard)
-              .map(m -> new MetacardImpl(m, m.getMetacardType()))
-              .map(ResultImpl::new)
-              .collect(Collectors.toList());
+          sourceResponse.getResults().stream().map(this::cloneResult).collect(Collectors.toList());
```

Comment:
Extracted the logic into its own private function below & added the copying of the relevancy+distance fields